services/udf/service.go

```diff
@@ -109,7 +111,7 @@ func (s *Service) Refresh(name string) error {
 }
 
 func (s *Service) loadUDFInfo(name string) (udf.Info, error) {
-	u, err := s.Create(name, s.logger, nil)
+	u, err := s.Create(name, "", "", s.logger, nil)
```

Comment:
Generally because the UDF is not being created in the context of a task or node.

More specificially because loadUDFInfo only makes the `Info` request and never sends the `Init` request so the taskID and nodeID will never be sent or subsequently used.

I should probably add a comment to that fact since its not clear.